deploy/contents/install/app/install-utils.sh

```diff
@@ -765,6 +771,13 @@ function parse_options {
     update_config_value "$CP_INSTALL_CONFIG_FILE" \
                         "CP_DEPLOYMENT_ID" \
                         "$CP_DEPLOYMENT_ID"
+
+    if  [ "$CP_DEPLOYMENT_TYPE" == "aws_eks" ] ; then
+        if [ -z "$CP_EKS_CSI_DRIVER_TYPE" ] || [ -z "$CP_EKS_CSI_EXECUTION_ROLE" ] || [ -z "$CP_EKS_CSI_FILESYSTEM_ID" ] ; then
```

Comment:
I think it would be better to rename `CP_EKS_CSI_FILESYSTEM_ID` to `CP_SYSTEM_FILESYSTEM_ID`

`CP_EKS_CSI_DRIVER_TYPE` -> `CP_KUBE_CSI_DRIVER_TYPE`
`CP_EKS_CSI_EXECUTION_ROLE` -> `CP_KUBE_CSI_EXECUTION_ROLE`